upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -581,19 +637,19 @@ metadata:
   name: karpenter
   namespace: kube-system
 ---
-# Source: karpenter/templates/webhook/deployment.yaml
+# Source: karpenter/templates/secret-webhook-cert.yaml
 apiVersion: v1
 kind: Secret
 metadata:
   name: karpenter-webhook-cert
   namespace: kube-system
 data: {} # Injected by karpenter-webhook
 ---
-# Source: karpenter/templates/100-config-logging.yaml
+# Source: karpenter/templates/configmap-logging.yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: karpenter-config-logging
+  name: config-logging
```

Comment:
This name is the default name for any knative logging config. It is way too generic for `kube-system`.